src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -393,4 +468,23 @@ export default class EnhancedReactTable<T> extends React.Component<IEnhancedReac
             ...this.state
         });
     }
+
+    private handleChangeItemsPerPage(itemsPerPage:number) {
+        const nextState = Object.assign({}, this.state);
```

Comment:
come to think of it I don't understand why this succeeds in typescript...it can't statically check this. This is just as deceptive to TS as casting. I'll change to casting.